### DISCUSSION TITLE
Chrome says Stylus will be unsupported – now what?

### DISCUSSION THREAD

**Discussion by Vendrel** [openstyles/stylus#1798] [General]
What to know about it? What can we do? (I don't want to use anything else but if there's nothing else we can do, I'd appreciate any tips on alternative plugins.) In which other browsers does the extension stay?

**Comment by tophf** [comment]
See https://github.com/openstyles/stylus/issues/1430#issuecomment-2131425173

> ManifestV2 can be enabled for a year via [policies](https://chromeenterprise.google/policies/) or [registry](https://www.reddit.com/r/browsers/comments/119x3qo/enabling_manifestv2_and_certain_external/) (if you're not an administrator, replace HKEY_LOCAL_MACHINE with HKEY_CURRENT_USER).
